gadget-container/gadgettracermanager/controller.go

```diff
@@ -31,7 +31,7 @@ import (
 
 	gadgetkinvolkiov1alpha1 "github.com/kinvolk/inspektor-gadget/pkg/api/v1alpha1"
 	"github.com/kinvolk/inspektor-gadget/pkg/controllers"
-	"github.com/kinvolk/inspektor-gadget/pkg/gadget-collection"
+	gadgetcollection "github.com/kinvolk/inspektor-gadget/pkg/gadget-collection"
```

Comment:
Can you explain the background of this change? what code smell are you trying to solve here?

Reply:
It was named by VS code, so i guess Go default formatter has changed. Anyway i found one more place (new commit). I'm glad to revert those two, but this should be problematic in the future.

Reply:
Why would it be problematic in the future?

Reply:
I mean the problem will be each user who uses Go format has to revert this change before every commit. Or everybody has to tweak formattter (if possible at all).

Reply:
Understood.  So, what you've done here is basically applying the latest go fmt to all files? If that's what you did, could you please re-word the commit stating that and including which version of go this was. It wasn't clear to me that this was the output of go fmt and thought that you were just applying your own personal preferences.

As a side note, we are aware that we need to get automatic formatting and automatic linting to get applied to all files when new versions come around, but we're still working on other automation tasks.

Reply:
I reverted them. I don't have time to figure out which default VS code feature did the formatting. I know one fact, i don't have any custom settings in VS Code :) Thank you for your review :)

Reply:
To avoid this kind of discussions in the future -> https://github.com/kinvolk/inspektor-gadget/issues/195